pygitguardian/sca_models.py

```diff
@@ -86,12 +90,13 @@ class SCAVulnerablePackageVersion(Base, FromDictMixin):
     vulns: List[SCAVulnerability] = field(default_factory=list)
 
 
-SCAVulnerablePackageVersion.SCHEMA = cast(
+SCAVulnerablePackageVersionSchema = cast(
     BaseSchema,
     marshmallow_dataclass.class_schema(
         SCAVulnerablePackageVersion, base_schema=BaseSchema
     )(),
```

Comment:
```suggestion
    ),
```
Not sure if this will change anything, but IAC schemas don't have these additional parenthesis

Reply:
Change this for all SCA schemas

Reply:
Just tested it, yes it does. `SCAVulnerablePackageVersionSchema` is not callable otherwise!